service/src/test/java/bio/terra/cbas/controllers/TestRunSetsApiController.java

```diff
@@ -177,30 +235,80 @@ void tooManyRecordIds() throws Exception {
     verifyNoInteractions(wdsService);
     verifyNoInteractions(cromwellService);
   }
-}
-
-class UnitTestRunSetsApiController {
-  private CbasApiConfiguration config = new CbasApiConfiguration();
-  private RunSetRequest request = new RunSetRequest();
 
   @Test
-  void testRequestValidityFewerThanMax() {
-    config.setRunSetsMaximumRecordIds(2);
-    request.setWdsRecords(new WdsRecordSet().recordIds(Arrays.asList("r1")));
-    assertTrue(RunSetsApiController.requestIsValid(request, config));
```

Comment:
This seems like maybe another case of an overly zealous rebase? It looks like this test has disappeared?

Reply:
Since I added back return error from `RunSetsApiController.requestIsValid` I had to refactor these tests too. They are kind of merged into these [2 tests here](https://github.com/DataBiosphere/cbas/blob/1a4fc18d77fc47a34cf5b66f6993f5c90e09bc18/service/src/test/java/bio/terra/cbas/controllers/TestRunSetsApiController.java#L279-L313). If the outcome of [this comment discussion](https://github.com/DataBiosphere/cbas/pull/60#discussion_r999949144) is to not return the error message to user yet I will revert my changes for the test as well.

Reply:
These tests have now been added back.